index.html

```diff
@@ -27,8 +27,19 @@ <h2>Sign up now for a free trial</h2>
         <input type="submit" value="Get started">
     </form>
 </section>
+<section id="reviews">
+    <h2>Reviews</h2>
+    <div class="review">
+        <img src="review1.jpg" alt="Review 1" class="round">
+        <p>"Daib.ai has been a game changer for me. The real-time error detection and code optimization has made coding a lot easier and quicker."</p>
+    </div>
+    <div class="review">
+        <img src="review2.jpg" alt="Review 2" class="round">
+        <p>"I love the web-based interface which allows me to access Daib.ai from anywhere. Highly recommend it!"</p>
+    </div>
+</section>
```

Comment:
Add a new section under the review section showing products that daib ai connects to like github, gitlab and bitbucket. Use any logo URL off the internet and position the logos side by side